documentation/features/DOCUMENTATIE-STANDAARD.md

```
@@ -0,0 +1,558 @@
+```
+
+#### Code Blocks
+```markdown
+{% code title="bestandsnaam.json" %}
+```json
+{
+  "example": "code"
+}
+```
+{% endcode %}
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)